src/components/pages/Contacts/index.tsx

```diff
@@ -234,6 +246,51 @@ const UnconnectedContacts: React.FC<PropsFromRedux> = ({
           </Modal.Footer>
         </Modal>
 
+        <Modal show={showSendModal}>
```

Comment:
think we could do show={!!selectedContact}

Reply:
good call - just made change